docs/linking/README.md

```diff
@@ -375,6 +379,25 @@ TODO!
 
 # 4. Linking Error Scenarios
 
+## 4.1. Discovery
+![Accounts error](../out/linking/error_scenarios/1-discovery-error.svg)
+
+## 4.2. Bad consentRequests
+When the DFSP receives the POST /consentRequests request from the PISP, any number of processing or validation errors could occur, such as:
+
+1. FSP does not support any requested authentication channels
```

Comment:
I would point out here error codes values for each error-case

Reply:
Ok error codes added